index.bs

```diff
@@ -480,7 +492,7 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
         absent, the implementer defined default mode for this {{contentType}} is
         assumed (i.e., the mode you get if you don't specify one via
         {{RTCRtpSender/setParameters()}}). scalabilityMode is only applicable
-        to the {{MediaEncodingType}} {{MediaEncodingType/webrtc}}.
+        to {{MediaEncodingConfiguration}} of the type {{MediaEncodingType/webrtc}}.
       </p>
 
       <p>
```

Comment:
nit: for spatialScalability (below), I'd go ahead and list out the types (file, rtc, ...) since this is the only one that doesn't mention them.

Reply:
Done!